statsd-client-async/statsd_client_core.ml

```diff
@@ -0,0 +1 @@
+../statsd-client/statsd_client_core.ml
```

Comment:
I'm not familiar with this syntax, what does this do? Pull in that file?

Reply:
symlinks to the file which async requires for building.